charts/linkerd-control-plane/templates/destination-rbac.yaml

```diff
@@ -230,3 +230,38 @@ subjects:
   - kind: ServiceAccount
     name: linkerd-destination
     namespace: {{.Release.Namespace}}
+---
+ apiVersion: rbac.authorization.k8s.io/v1
+ kind: ClusterRole
+ metadata:
+   name: linkerd-policy-status
+   labels:
+     app.kubernetes.io/part-of: Linkerd
+     linkerd.io/control-plane-component: destination
+     linkerd.io/control-plane-ns: {{.Release.Namespace}}
+     {{- with .Values.commonLabels }}{{ toYaml . | trim | nindent 4 }}{{- end }}
+ rules:
+   - apiGroups:
+       - policy.linkerd.io
+     resources:
+       - httproutes/status
+     verbs:
+       - patch
```

Comment:
is there any reason this should be separate from the existing policy RBAC?

Reply:
Nope good catch I consolidated it to a separate rule in the existing RBAC.